addons/l10n_latam_check/tests/test_own_checks.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.l10n_latam_check.tests.common import L10nLatamCheckTest
+from odoo.tests.common import Form, tagged
+from odoo import fields
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestOwnChecks(L10nLatamCheckTest):
+
+    def test_01_pay_with_manual_check(self):
+        """ Create and post a manual check with deferred date """
+
+        with Form(self.env['account.payment'].with_context(default_payment_type='outbound')) as payment_form:
+            payment_form.ref = 'Deferred check'
+            payment_form.l10n_latam_check_payment_date = fields.Date.add(fields.Date.today(), months=1)
```

Comment:
maybe this one needs to be done after setting the payment method.  This is not the most worthwhile test anyways...  @zaoral

Reply:
Hi @jco-odoo 

Indeed. I also think it needs to be done after setting the payment method.

However, I do not know why but I am not able to reproduce this error in my local environment. I have the last updated odoo and enterprise repos 16.0. Maybe there is something I am doing wrong? leave you here an screenshot showing the unit test run and the command I use 
![test-local](https://user-images.githubusercontent.com/7593953/205299793-bd02290a-5f51-4263-8436-af63580a1b5b.png)

I was not able to test it, but I have applied the change you suggested and push the changes. Please could you run the L10n Stage for the last push and we wait for the result